scripts/dev-e2e.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+
+GREEN='\033[0;32m'
```

Comment:
```suggestion
RED='\031[0;32m'
GREEN='\033[0;32m'
```

Reply:
Good suggestion - `RED` has been added in commit 57c8852, and errors are rendered with it. I made a new func `logerr`, as it seemed verbose to have every call to a more general func also specify a color/type parameter, but I'm open to changing that.